apps/wallet-connect/src/hooks/useWalletConnect.tsx

```diff
@@ -56,93 +59,8 @@ const useWalletConnect = () => {
         }
 
         try {
-          let result = '0x'
-
-          switch (payload.method) {
-            case 'eth_sendTransaction': {
-              const txInfo = payload.params[0]
-              const { safeTxHash } = await sdk.txs.send({
-                txs: [
-                  {
-                    to: txInfo.to,
-                    value: txInfo.value || '0x0',
-                    data: txInfo.data || '0x',
-                  },
-                ],
-              })
-
-              result = safeTxHash
-              break
-            }
-            case 'gs_multi_send': {
```

Comment:
I supose this was a test method that wasn't still implemented?

Reply:
This is the method for batching transactions implemented by the OG app. I'm not sure if it's widely used. Quick GitHub search shows 1-2 apps: https://github.com/search?q=gs_multi_send&type=code

I think the mobile app continues to support it, so if we want to keep the support, we can implement it in the provider